src/core/index.tsx

```diff
@@ -1,14 +1,19 @@
 import React, { FC } from 'react';
+import { Helmet, HelmetProvider } from 'react-helmet-async';
 import Footer from './components/Footer/index';
 import Header from './components/Header/index';
 import './less/core.less';
 
 const Core: FC = ({ children }) => (
-  <>
+  <HelmetProvider>
+    <Helmet>
+      <title>Linjeforeningen Online</title>
+    </Helmet>
+
     <Header />
     <main>{children}</main>
     <Footer />
-  </>
+  </HelmetProvider>
```

Comment:
Adding the Provider again here will probably make Helmet not work in SSR?

When using a context the closest provider above in the tree will be used, which would be this provider in all cases.

If a provider needs a different configuration for client and server it should be added to the `src/server/index.tsx` file and the `src/index.tsx` file.

Reply:
So I refactored it, by moving the `<HelmetProvider>` to `<ContextWrapper>` (which i think makes the most sense), and adding the `HelmetContext` as an optional input, and that works both with SSR and without locally for me